examples/squeezenet/SqueezeNet.js

```diff
@@ -68,12 +76,17 @@ class SqueezeNet {
   _addTensorOperands() {
     const graph = this._onnxModel.graph;
     for (let i = 0; i < graph.input.length; ++i) {
+      // delete useless empty node in the graph
+      if (graph.input[i].type.tensorType.elemType === 7 && graph.input[i].type.tensorType.shape.dim.length === 0) {
```

Comment:
What is `elemType` as `7`? Could you define a named constant or some constant defined in the onnx.js?

Reply:
`7` stands for `onnx.TensorProto.DataType.INT64`. It's a dummy node in the onnx graph and this is just a workaround for supporting onnx mobilenet. Later, I will try to remove these lines.

Reply:
If you want to keep this line, please use `onnx.TensorProto.DataType.INT64`, then it will be fine.